TCAT/Controllers/RouteDetailDrawerViewController+Extensions.swift

```diff
@@ -90,9 +90,16 @@ extension RouteDetailDrawerViewController: PulleyDrawerViewControllerDelegate {
 }
 
 extension RouteDetailDrawerViewController: UITableViewDataSource {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return sections.count
+    }
 
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return directionsAndVisibleStops.count
+        switch sections[section] {
+        case .routeDetail: return directionsAndVisibleStops.count
+        case .notification: return notificationTitles.count
```

Comment:
Instead of storing the associated arrays on the viewController (`.routeDetail -> directionsAndVisibleStops` and `.notification -> notificationTitles`), lets give each of the enum options their own associated values. So `.routeDetail([RouteDetailItem])` and `.notification([String])`